check_stability.py

```diff
@@ -427,7 +427,9 @@ def write_inconsistent(inconsistent, iterations):
 def write_results(results, iterations):
     logger.info("## All results ##\n")
     for test, test_results in results.iteritems():
-        logger.info("### %s ###" % test)
+        baseurl = "https://w3c-test.org/submissions"
+        pr = os.environ['TRAVIS_PULL_REQUEST']
```

Comment:
This case be `"false"`, which is probably not what you want. Pass through args.comment_pr and check that it parses as an int.